src/goto-diff/unified_diff.cpp

```diff
@@ -396,3 +396,8 @@ void unified_difft::output(std::ostream &os) const
       os);
   }
 }
+
+const unified_difft::differences_mapt &unified_difft::differences_map() const
+{
+  return differences_map_;
+}
```

Comment:
Would it be worth keeping that in the header file? I actually like what you've done here better, but most other code seems to leave such simple getters in the class declaration.

Reply:
There's not much in it. It's more locally consistent to organize it this way, so I'd be inclined to keep it, especially in the absence of a hard rule in the coding standard that says where to place short methods.

I'm leaving it for now, but if you'd prefer it to go in the header I'm happy to change it too.

Reply:
Has the use of a "p_" or "m_" prefix been considered? I accept that this is all a matter of taste, and thus it's probably just me whose eyes hurt when seeing this.